contents/docs/integrate/client/js/index.mdx

```diff
@@ -265,58 +268,88 @@ function signup(email) {
 }
 ```
 
+## Persistence
+
+In order for PostHog to work optimally, we require storing a small amount of information about the user on the user's browser. This ensures that if the user navigates away, and comes back to your site at a later time, we will still identify them properly. We store the following information in the user's browser:
+
+-   User's ID
+-   Session ID & Device ID
+-   Active & enabled feature flags
+-   Any super properties you have defined.
+-   Some PostHog configuration options (e.g. whether session recording is enabled)
+
+By default we store all this information in a `cookie`, however due to the size limitation of cookies, you may run into some issues (e.g. if you have a lot of feature flags). You can use any of the following options to store this information by setting the `persistence` parameter in PostHog's config.
+
+-   `localstorage+cookie`. User's distinct ID is stored in a cookie and everything else is stored in the browser's <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage" target="_blank">localStorage</a>.
```

Comment:
@timgl I couldn't find any strong reference that signals that `localStorage` is less persistent than cookies. Seems like both are kept until the user deletes them (or expiration on cookies), but if you know of any trade-off we can document it here.

Reply:
The tradeoff is that you can't use localStorage across subdomains, so you could get into a situation where we track the user across but any superproperties are incorrect.

https://stackoverflow.com/questions/4026479/use-localstorage-across-subdomains